Updating the Auto Annotation section of the CVAT User Guide (#996)
opencv/cvat: cvat/apps/documentation/user_guide.md

![](static/documentation/images/gif009_DETRAC.gif)

## Automatic annotation

> Automatic Annotation is used for creating preliminary annotation. 

Reviewer: ```suggestion
Automatic Annotation is used for creating preliminary annotations. 
```

To use Automatic Annotation you need a DL model. You can use primary models or models uploaded by a user. 
You can find the list of available models in the ``Models`` section.

1.  To launch automatic annotation, you should open the dashboard and find a task which you want to annotate. 
    Then click the ``Actions`` button and choose option ``Automatic Annotation`` from the dropdown menu.

    ![](static/documentation/images/image119_DETRAC.jpg)

> 1.  In dialogue window select a model you need. DL models are created for specific labels, e.g. 

Reviewer: ```suggestion
1.  In the dialog window select a model you need. DL models are created for specific labels, e.g. 
```

>     the Crossroad model was created whit using footage from cameras located above the highway, 

Reviewer: ```suggestion
    the Crossroad model was created with using footage from cameras located above the highway, 
```

>     the Crossroad model was created was taught using footage from cameras located above the highway, 

Reviewer: Need to fix:
"was created was taught"

>     which is you should use this model with the tasks with similar camera angles. 

Reviewer: ```suggestion
    which you should use this model with the tasks with similar camera angles. 
```
